src/Symfony/Component/Intl/composer.json

```diff
@@ -24,11 +24,11 @@
         }
     ],
     "require": {
-        "php": ">=8.2"
+        "php": ">=8.2",
+        "symfony/emoji": "^7.0"
```

Comment:
this should be removed - we don't want to make ext-intl a mandatory dep of symfony/intl
we can ease discoverability by throwing a logic exception when the legacy EmojiTransliterator is missing the new component

Reply:
Still navigating the BC world in my head (slow learner 😅): if we don't guarantee that symfony/emoji is installed, a project currently using the EmojiTransliterator will break after a composer update, no ?

Reply:
There are two aspects to BC here:

1. this is the BC promise the following, and you'll not mention of deps there: https://symfony.com/doc/current/contributing/code/bc.html
2. requiring an extension is another BC break from another PoV - could be a worse one since it requires a change to the infrastructure

We already did similar changes in the past. Yes, we prefer doing things as smooth as possible, but adding a required extension doesn't fit.

Reply:
Yep ok i understand (not all yet, but more than 12mn ago 😃 )